src/components/icon/img/ui5-newscast.svg

```diff
@@ -1 +1 @@
lipPath></defs><path class="B" d="M18 12.32V9.87a4 4 0 0 1-2.16-.59A1.75 1.75 0 0 1 15 7.81V4.7a1.76 1.76 0 0 1 .87-1.45 3.54 3.54 0 0 1 2.13-.6h7.4a3.49 3.49 0 0 1 2.07.61 1.75 1.75 0 0 1 .88 1.45v3.1a1.74 1.74 0 0 1-.88 1.47 3.62 3.62 0 0 1-2.07.59h-5.3l-2 4zm-1.55 13.23a6.25 6.25 0 0 1 2.46.5A6.41 6.41 0 0 1 21 27.42a6.54 6.54 0 0 1 1.37 2 6.26 6.26 0 0 1 .49 2.47v6.4H3.7v-6.37a6.27 6.27 0 0 1 .5-2.47 6.37 6.37 0 0 1 1.37-2 6.51 6.51 0 0 1 2-1.37 6.19 6.19 0 0 1 2.49-.5h6.38zm3.2-6.38a6.19 6.19 0 0 1-.5 2.49 6.41 6.41 0 0 1-5.88 3.89 6.19 6.19 0 0 1-2.49-.5 6.48 6.48 0 0 1-3.39-3.39 6.19 6.19 0 0 1-.5-2.49 6.25 6.25 0 0 1 .5-2.46 6.41 6.41 0 0 1 1.37-2.05 6.56 6.56 0 0 1 2-1.37 6.19 6.19 0 0 1 2.49-.5 6.34 6.34 0 0 1 2.47.5 6.54 6.54 0 0 1 2 1.37 6.41 6.41 0 0 1 1.37 2.05 6.25 6.25 0 0 1 .56 2.46z"/><path class="C" d="M32.8 30.47a7.76 7.76 0 0 1 3.09.63 7.88 7.88 0 0 1 4.27 4.27 7.71 7.71 0 0 1 .62 3.08v8H16.85v-8a7.72 7.72 0 0 1 .63-3.08 7.88 7.88 0 0 1 1.71-2.56 8.13 8.13 0 0 1 2.53-1.71 7.62 7.62 0 0 1 3.11-.63h8zm4-8a7.66 7.66 0 0 1-.62 3.12 8.31 8.31 0 0 1-1.71 2.53 8 8 0 0 1-2.56 1.71 7.75 7.75 0 0 1-3.09.62 7.61 7.61 0 0 1-3.11-.62 8.13 8.13 0 0 1-2.53-1.71 8 8 0 0 1-1.71-2.53 7.94 7.94 0 0 1 0-6.2 7.88 7.88 0 0 1 1.71-2.56 8 8 0 0 1 2.53-1.71 7.62 7.62 0 0 1 3.11-.63 7.76 7.76 0 0 1 3.09.63 7.88 7.88 0 0 1 2.56 1.71 8.21 8.21 0 0 1 1.71 2.56 7.67 7.67 0 0 1 .61 3.1z"/><path class="D" d="M16.84 40.94v-2.36a8.93 8.93 0 0 1 .47-2.81h23a8.53 8.53 0 0 1 .43 2.12v3.2h-23.9z"/><path class="E" d="M40.35 35.77h-23s1.66-6.26 10.12-5.3h5.84a7.78 7.78 0 0 1 7.04 5.3z"/><circle class="F" cx="28.76" cy="22.46" r="8.04"/><g clip-path="url(#A)"><path class="C" d="M36.14 20.9A8 8 0 0 1 28 17.36a7.75 7.75 0 0 1-1.16-2.92l-1.4-7.86L49 2.36l1.4 7.86a7.72 7.72 0 0 1-.07 3.14 8.11 8.11 0 0 1-3.42 5 7.88 7.88 0 0 1-3 1.16l-3.93.7z"/><path class="D" d="M50 7.77l.42 2.32a9.19 9.19 0 0 1 0 2.85L27.8 17a8.81 8.81 0 0 1-.8-2l-.58-3.16L50 7.6z"/><path class="E" d="M27.8 17l22.7-4.06s-.54 6.46-9 7l-5.74 1A7.79 7.79 0 0 1 27.79 17z"/></g><g clip-path="url(#B)"><path class="C" d="M39.43 18.43a7.94 7.94 0 0 1-3.15-.07 8 8 0 0 1-5-3.46 7.75 7.75 0 0 1-1.12-2.9l-1.4-7.86L52.32-.1l1.4 7.86a7.72 7.72 0 0 1-.07 3.14 8.11 8.11 0 0 1-3.42 5 7.88 7.88 0 0 1-3 1.16l-3.93.7z"/><path class="D" d="M53.3 5.3l.42 2.32a9.19 9.19 0 0 1 0 2.85l-22.65 4.06a8.81 8.81 0 0 1-.8-2l-.57-3.16 23.57-4.23z"/><path class="E" d="M31.07 14.54l22.7-4.06s-.54 6.46-9 7l-5.74 1a7.77 7.77 0 0 1-7.95-3.94z"/></g><g clip-path="url(#C)"><path class="C" d="M33.23,23.53a8,8,0,0,1-3.15-.07,8.13,8.13,0,0,1-2.82-1.24A8.34,8.34,0,0,1,25.12,20,7.85,7.85,0,0,1,24,17.08L22.56,9.22,46.13,5l1.4,7.86A7.77,7.77,0,0,1,47.46,16a7.92,7.92,0,0,1-1.23,2.82A8.22,8.22,0,0,1,44,21a7.65,7.65,0,0,1-3,1.16l-3.92.7Z"/><path class="D" d="M47.1 10.4l.4 2.33a8.78 8.78 0 0 1 0 2.84l-22.65 4.06a9 9 0 0 1-.79-2l-.58-3.16 23.58-4.22z"/><path class="E" d="M24.87 19.64l22.7-4.06s-.53 6.46-9 7l-5.74 1a7.8 7.8 0 0 1-7.95-3.94z"/></g></svg>
```

Comment:
Though, the viewport of this icon is fine, but fractional pixels inside are overlapping and produce an unpleasant noise (hardness) effect. This has be to fixed by the designer:
![1](https://user-images.githubusercontent.com/748043/53340296-f4238180-3908-11e9-9fcc-0d30d76be8b3.png)
![2](https://user-images.githubusercontent.com/748043/53340428-4cf31a00-3909-11e9-839a-074599cbe72f.png)
![3](https://user-images.githubusercontent.com/748043/53340299-f4238180-3908-11e9-89b7-7a2d67fe4793.png)